backend/app/api/admin_routes/user.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter, Depends
+from fastapi_pagination import Params, Page
+from fastapi_pagination.ext.sqlmodel import paginate
+from sqlmodel import select
+
+from app.api.deps import SessionDep, CurrentSuperuserDep
+from app.models import User
+router = APIRouter()
+
+
+@router.get("/admin/users")
+def list_users(
+    session: SessionDep,
+    user: CurrentSuperuserDep,
+    params: Params = Depends(),
+) -> Page[User]:
+    return paginate(
+        session,
+        select(User).order_by(User.id),
```

Comment:
How about:
- `/admin/users` -> `/admin/users/search`
- `list_users` -> `search_users`

Maybe we need a `search`  parameter for user search. The user list may be very long, and we need to consider how to help users find the user_id they need.

For the return type, you can using `UserDescriptor` instead to avoid return unnecessary or sensitive information  to the frontend.

Add `email: str` to `UserDescriptor`:

https://github.com/pingcap/autoflow/blob/9b05f1016c0df9b6bca18ba7ad01c3b7579bab64/backend/app/api/admin_routes/models.py#L21C7-L21C21